package.json

```diff
@@ -15,69 +15,71 @@
   },
   "dependencies": {
     "@auth0/nextjs-auth0": "^3.5.0",
-    "@buf/safedep_api.bufbuild_es": "^1.10.0-20241230154048-f72c3643452e.1",
-    "@buf/safedep_api.connectrpc_es": "^1.6.1-20241117092917-2fd4dbf7c52e.1",
+    "@buf/safedep_api.bufbuild_es": "1.10.0-20250127063129-a4e14b7d4f45.1",
```

Comment:
Why remove `^`? Pinning to specific version makes it difficult to upgrade. All other dependencies have a range constraint and not static version